lms/templates/user_dropdown.html

```diff
@@ -97,4 +97,8 @@
             </div>
         </li>
     </ol>
+    <script>
+        var realUserName = '${self.real_user.get_full_name()}';
```

Comment:
I would prefer if you set data into js through the `data-` attributes and not embedding `<script>` into a template.